src/main/scala/diplomaticobjectmodel/DiplomaticObjectModelUtils.scala

```diff
@@ -125,18 +125,19 @@ object DiplomaticObjectModelAddressing {
     )
   }
 
-  private def omAddressSets(ranges: Seq[AddressSet]): Seq[OMAddressSet] = {
+  private def omAddressSets(ranges: Seq[AddressSet], name: String): Seq[OMAddressSet] = {
     ranges.map {
       case AddressSet(base, mask) =>
+        require(mask != 0, s"""ERROR: omAddressSets: $name is mask is 0""")
```

Comment:
"name is mask is 0" seems like a typo. Also, putting the `ERROR` at the beginning is not necessary because this very clearly causes a requirement failure error in Scala, and it also doesn't match our assertion message style.

```suggestion
        require(mask != 0, s"omAddressSets: $name has invalid mask of 0")
```

Reply:
fixed